backend/src/resolvers/posts.js

```diff
@@ -1,6 +1,43 @@
-import { neo4jgraphql } from 'neo4j-graphql-js'
+import { neo4jgraphql, cypherQuery } from 'neo4j-graphql-js'
+import { v1 as neo4j } from 'neo4j-driver'
+import _ from 'lodash'
+
+export function extractQueryResult ({ records }, returnType) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T23:24:08Z" title="Friday, May 3rd 2019, 1:24:08 am +02:00">May 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,43 @@
-import { neo4jgraphql } from 'neo4j-graphql-js'
+import { neo4jgraphql, cypherQuery } from 'neo4j-graphql-js'
+import { v1 as neo4j } from 'neo4j-driver'
+import _ from 'lodash'
+
+export function extractQueryResult ({ records }, returnType) {
```


All of this is copy+paste from `neo4j-graphql-js`.